pcdsdevices/epics_motor.py

```diff
@@ -200,39 +203,28 @@ def check_value(self, value):
         """
         Check if the motor is disabled
         Check if the value is within the soft limits of the motor.
+        Check if the spg field is on "pause" or "stop"
 
         Raises
         ------
-        Exception
+        MotorDisabledError(MoveForbiddenError)
             If the motor is passed any motion command when disabled
-        ValueError
+        LimitError(ValueError)
             When the provided value is outside the range of the low
             and high limits
+        MotorSPGError(MoveForbiddenError)
```

Comment:
Not sure we need this much resolution. I'm kind of fine with a `MotorDisabledError` for both cases

Reply:
I opted for high resolution so that other code could decide how to enable the motor (call `go()` or `enable()`), but perhaps this is unnecessary because you can just call both with no ill effects.

As an aside, having an `enable` that doesn't set `go` makes me slightly uncomfortable, but I'm not sure what to do better here.